cmd/server/Dockerfile

```diff
@@ -20,6 +20,8 @@ COPY --from=build /app/blood-info blood-info
 COPY --from=build /app/tls-self-signed-cert tls-self-signed-cert
 
 # FIXME: This is a hack to get the self-signed certificate to work. There must be a better way.
+# FIXME: This makes the secret `key.pem` also part of the image; uploading the image anywhere would leak it!
+# FIXME: any prior `cert.pem` & `key.pem` you had in current dir are *also* leaked in the COPY layer.
```

Comment:
Created https://github.com/il-blood-donation-info/blood-donation-backend/issues/37 to track this